Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -3882,6 +3977,86 @@ def handle_incoming_resolved_issue(issue: Dict[str, Any]) -> Dict[str, Any]:
     return closing_entry
 
 
+def issue_get_forms_command(client: JiraBaseClient, args: Dict[str, Any]) -> List[CommandResults]:
+    """Pulls down all forms with their questions and answers for a given issue
+
+    :param client: The Jira client to use for making the API calls
+    :type client: JiraBaseClient
+    :param args: Generic arguments dict which has the argument `issue_id` for finding
+                 the specific issue and it's forms
+    :type args: Dict[str, Any]
+    :raises DemistoException: When the command is tried for a Jira Cloud platform which is not supported.
+    :raises ValueError: When the `issue_id` argument is not supplied
+    :return: One CommandResult per form that is found with the form data
+    :rtype: List[CommandResults]
+    """
+    if not isinstance(client, JiraOnPremClient):
+        raise DemistoException('This command is only supported on Jira OnPrem')
+
+    issue_id = args.get('issue_id', '')
+    if not issue_id:
+        raise ValueError('No issue_id specified for jira-get-issue-forms')
+
+    raw, forms = get_issue_forms(client, issue_id)
+    if not forms:
+        return CommandResults(readable_output="No forms found")
+
+    results = []
+    for form in forms:
+        results.append(CommandResults(
+            outputs_prefix='Jira.Forms',
+            outputs_key_field='ID',
+            outputs=form,
+            readable_output=f'Pulled data for form {form.get("ID")} from issue {issue_id}.',
+            raw_response=raw
+        ))
+    return results
+
+
+def get_user_command(client: JiraBaseClient, args: Dict[str, Any]) -> CommandResults:
+    """Gets a user's information from Jira
+
+    :param client: The Jira client for calling the API
+    :type client: JiraBaseClient
+    :param args: Generic arguments dict which has the argument `username` or `key` for finding
+                 the user
+    :type args: Dict[str, Any]
+    :raises ValueError: When no key or username is provided to the command
+    :return: The CommandResults object with the data returned by the API
+    :rtype: CommandResults
+    """
+    key = args.get('key', '')
+    username = args.get('username', '')
+    if key:
+        identifier = f'key={key}'
+    elif username:
+        identifier = f'username={username}'
```

Comment:
This parameter is no longer available. See the [deprecation notice](https://developer.atlassian.com/cloud/jira/platform/deprecation-notice-user-privacy-api-migration-guide) for details.

Reply:
So that doesn't apply to on prem/datacenter as there is no Atlassian ID for users. Should I move this command to the on prem class then?

Reply:
Fixed, cloud clients will only use accountId and on prem will use the legacy params